hudi-common/src/main/java/org/apache/hudi/common/table/read/HoodieBaseFileGroupRecordBuffer.java

```diff
@@ -79,25 +81,21 @@ public abstract class HoodieBaseFileGroupRecordBuffer<T> implements HoodieFileGr
   protected InternalSchema internalSchema;
   protected HoodieTableMetaClient hoodieTableMetaClient;
 
-  public HoodieBaseFileGroupRecordBuffer(HoodieReaderContext<T> readerContext,
-                                         HoodieTableMetaClient hoodieTableMetaClient,
-                                         Option<String> partitionNameOverrideOpt,
-                                         Option<String[]> partitionPathFieldOpt,
-                                         HoodieRecordMerger recordMerger,
-                                         TypedProperties props) {
+  public HoodieBaseFileGroupRecordBuffer(HoodieReaderContext<T> readerContext, HoodieReaderState readerState) {
```

Comment:
Might be good to have another PR to strip `Hoodie` prefix from the file group reader classes.

Reply:
It's already under `org.apach.hudi` package, and the file group concept is Hudi-specific already; adding `Hoodie` prefix in the class name makes it redundant.

Reply:
https://issues.apache.org/jira/browse/HUDI-8171